### PR TITLE
Required changes to be compatible with GLPI 9.2

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: phpunit/RestoreDatabase_TestCase.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<?php
+n<?php
```
Comment (orthagh) [inline phpunit/RestoreDatabase_TestCase.php]: !

Comment (trasher) [inline phpunit/RestoreDatabase_TestCase.php]: Erf...

**Comment by trasher** [comment]
As for the frequencies, yes, this is mandatory. A PR have been merge on the 9.1 branch to drop this extra data in the FormatConvert class, but unit tests are populating directly data.
Also, with the " MHz", we got a warning with PHP 7.1 from the core.

**Comment by orthagh** [comment]
ok fine for me :)

**Pull request merged**